routes/v0/csam-reports.js

```diff
@@ -0,0 +1,19 @@
+const models = require('../../models');
+const { SafeRouter, publicEndpoint } = require('../../permissions');
+
+const { CSAMReport } = models;
+const CSAMReportController = require('../../controllers/csam-report-controller')(CSAMReport);
+
+const csamReportRouter = SafeRouter();
+
+csamReportRouter.post('/', publicEndpoint, async (
```

Comment:
Do we need to set up permissions for this endpoint using our permissions framework?

Reply:
I gave to this one the same permissions as creating a contact, should it be different? I assume counselors will be the ones performing this action the most.

Reply:
Same permissions as creating a contact sounds reasonable. Where is that coded?